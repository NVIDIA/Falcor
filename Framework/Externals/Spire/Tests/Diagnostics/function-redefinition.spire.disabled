// redefining a function

int foo(int a) { return 0; }
int foo(int b) { return 1; }
