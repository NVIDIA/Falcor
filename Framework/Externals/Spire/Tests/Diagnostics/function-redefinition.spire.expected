result code = -1
standard error = {
Tests/Diagnostics/function-redefinition.spire(4): error 30001: 'foo(int)': function redefinition.
}
standard output = {
}
